services/motion/builtin/move_request.go

```diff
@@ -655,6 +665,9 @@ func (ms *builtIn) relativeMoveRequestFromAbsolute(
 	startPoseInv := spatialmath.PoseInverse(startPose.Pose())
 
 	goal := referenceframe.NewPoseInFrame(referenceframe.World, spatialmath.PoseBetween(startPose.Pose(), goalPoseInWorld))
+	if spatialmath.PoseAlmostCoincidentEps(goal.Pose(), spatialmath.NewZeroPose(), motionCfg.planDeviationMM) {
+		return nil, errors.Errorf("no need to move, already within planDeviationMM: %f", motionCfg.planDeviationMM)
```

Comment:
[opt] Not a huge deal but would be in favor of making this a function in `errors.go`. If you do the test you write will be a little bit more robust to changes in the string

Reply:
i am in favor of this, do you think I should put this in `spatialmath/errors.go`? alternatively I could put this as an error func at the top of the `move_request.go` file.

Reply:
Actually we're really inconsistent about doing this and its so negligible in terms of utility.  I think its fine as is